src/edge/simulator.ts

```diff
@@ -226,36 +228,34 @@ export class Simulator {
     }
 
     private async getCurrentSimulatorVersion(): Promise<string | undefined> {
-        const output: string = await Executor.executeCMD(undefined, this.getSimulatorExecutorPath(true), { shell: true }, "--version");
+        const output: string = await Executor.executeCMD(undefined, this.getAdjustedSimulatorExecutorPath(true), { shell: true }, "--version");
         const version: string | null = Simulator.extractVersion(output);
         return version;
     }
 
     private async simulatorInstalled(): Promise<SimulatorType> {
-        const exist = await Simulator.checkCmdExist(this.getSimulatorExecutorPath(true));
-        if (exist) {
-            return Simulator.currentPlatform === "win32" ? SimulatorType.Standalone : SimulatorType.Pip;
-        } else {
-            return SimulatorType.NotInstalled;
+        if (this.standaloneSimulatorExecutablePath) {
+            const exist = await Simulator.checkCmdExist(this.getAdjustedSimulatorExecutorPath(true));
```

Comment:
check the path before calling checkCmdExist?

Reply:
OK, check it inside checkCmdExist